run_ingestion/common/pubsub_publisher.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from google.cloud import pubsub_v1
+
+
+def notify_data_ingested(id, gcs_bucket, filename):
+  """Publishes a notification on the notify-data-ingested topic indicating that
+     some data was ingested.
+
+     id: The id of the source that was ingested
+     gcs_bucket: The name of the bucket the data is located in
+     filename: The name of the file that was uploaded"""
+  topic = 'notify-data-ingested'
+  notify_topic(topic, id=id, gcs_bucket=gcs_bucket, filename=filename)
+
+
+def notify_topic(topic, **attrs):
+  """Publishes a notification on the specified topic using the provided
+     attributes
+
+     topic: The name of the topic to notify on
+     attrs: The attributes to pass through to the message"""
+  publisher = pubsub_v1.PublisherClient()
+  # For some reason topic_path doesn't show up as a member, but is used in the
+  # official documentation:
+  # https://googleapis.dev/python/pubsub/latest/publisher/api/client.html?highlight=topic_path#google.cloud.pubsub_v1.publisher.client.Client.publish
+  # pylint: disable=no-member
+  topic_path = publisher.topic_path('fellowship-test-internal', topic)
```

Comment:
This needs to be changed to the sandbox environment. Or better yet, passed in via terraform. Probably best to do this as a separate PR though. I can take this on.

Feel free to hard code this to the sandbox environment or just leave it, your call